subsys/spm/secure_services.c

```diff
@@ -114,13 +109,13 @@ void spm_request_system_reboot_nse(void)
 __TZ_NONSECURE_ENTRY_FUNC
 int spm_request_random_number_nse(uint8_t *output, size_t len, size_t *olen)
 {
-	int err;
+	int err = 0;
```

Comment:
For safety, this value should be set to an error-value in case this code changes and there is a fall-through that is not catched